weave/tests/test_execute_fast.py

```diff
@@ -27,3 +28,30 @@ def test_resolve_static_branches():
     d = weave.save({"a": 5})
     l = weave.save([1, 2])
     assert weave.use(l.map(lambda row: row + d["a"])) == [6, 7]
+
+
+def test_empty_list():
```

Comment:
this is a pretty wild test for what we are testing ... does the map function need this much complexity?

Reply:
simplified